### PR TITLE
Make it easier for users to test spaceship

### PR DESCRIPTION
**Pull request by KrauseFx**
Let's just include the content by default

### CODE REVIEW

File: ToolsAndDebugging.md [hunk 1]
```diff
@@ -162,24 +162,31 @@ SPACESHIP_DEBUG=1 spaceship
...
   apps = Spaceship::Tunes::Application.all
+
   require 'pry'
   binding.pry
+
+  puts "All done"
 end
 ```
 
+Optionally, add `Spaceship.select_team` right after the `.login` call, if you're part of multiple teams
```
Comment (KrauseFx) [inline ToolsAndDebugging.md]: I think there are 2 different team selection methods, right? Depending on if iTC or Portal

Comment (milch) [inline ToolsAndDebugging.md]: Hmm, then which one gets called when you just run `Spaceship.select_team` ?

Comment (milch) [inline ToolsAndDebugging.md]: Just checked, it's an alias for `Spaceship::Portal.select_team`. In that case this needs to be updated to tell users to use the correct one 👍

**Comment by ohwutup** [comment]
Hey @KrauseFx and @milch! I am going to close this for now as the monogem work removed the `Rakefile` from _spaceship_ and I figured this would be easy to copy over to a better place (I just didn't know where that better place would be right now 😛 )

**Comment by KrauseFx** [comment]
Great point @ohwutup 👍

**Pull request closed without merging**